test/common/page_objects/add-job-page.js

```diff
@@ -14,6 +14,7 @@ const AddJobPage = function AddJobPage(browser) {
   this.getValidationError = () => browser.text('#validation-errors');
 
   this.visit = (accountId) => browser.visit(`/${accountId}/jobs/new`);
+  this.patch = (accountId, jobId) => browser.visit(`/${accountId}/jobs/${jobId}?_method=PATCH`);
```

Comment:
Ah of course, sorry I should have realised.

I think its worth adding an integration test for the change to the patch method, its the sort of behaviour that is just non-obvious enough for us to forget about